From: Nico Golde <nion@debian.org>
Description: Remove version and upstream specific notes from manual page
diff -urNad tcpxtract-1.0.1~/tcpxtract.1 tcpxtract-1.0.1/tcpxtract.1
--- tcpxtract-1.0.1~/tcpxtract.1	2005-10-06 03:53:57.000000000 +0200
+++ tcpxtract-1.0.1/tcpxtract.1	2009-10-01 19:04:07.000000000 +0200
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.35.
 .TH TCPXTRACT "1" "October 2005" "tcpxtract v1.0.1" "User Commands"
 .SH NAME
-tcpxtract \- manual page for tcpxtract v1.0.1
+tcpxtract \- extract files from captured network packets
 .SH SYNOPSIS
 .B tcpxtract
 [\fIOPTIONS\fR] [[\fI-d <DEVICE>\fR] [\fI-f <FILE>\fR]]
@@ -25,15 +25,3 @@
 .TP
 \fB\-\-help\fR, \fB\-h\fR
 display this lovely screen
-.SH "SEE ALSO"
-The full documentation for
-.B tcpxtract
-is maintained as a Texinfo manual.  If the
-.B info
-and
-.B tcpxtract
-programs are properly installed at your site, the command
-.IP
-.B info tcpxtract
-.PP
-should give you access to the complete manual.
